clients/python/sysadminctl

```diff
@@ -89,7 +89,26 @@ class SetCommand(object):
             pyvalue = convertToType(args.value, existing_type)
         else:
             pyvalue = convertToType(args.value, args.type)
-        return format_set_output(args.key, client.set(args.key, pyvalue))
+        return format_set_output(client.set(args.key, pyvalue))
+
+
+class ModifyCommand(object):
+    def __init__(self, argparser):
+        self.parser = argparser.add_parser("modify")
+        self.parser.add_argument("key", type=str,
+                                 help="key")
+        self.parser.add_argument("value", type=str, nargs="*",
+                                 help="value")
+
+    def run(self, args):
+        client = SysAdminClient(args.host, args.port, args.xid)
+        existing = client.get(args.key)
+        existing_type = get_existing_type(existing)
```

Comment:
I may have forgotten how this even works, but why fetch the type when you can just check if the value exists?

Reply:
To force them to be a bit more explicit about what they're doing. The command itself is basically training wheels